README.md

```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images hosted on Docker Hub. The main tenets are:
+
```

Comment:
Thoughts on adding one more at the top here?

```suggestion

- Focus on [Free](https://www.fsf.org/) and [Open-Source](https://opensource.org/) Software

```

Reply:
Seems like a fine one to add, but should it be first? (I guess the real question, how are they ordered?)

Reply:
I think the order is somewhat random ATM (maybe @wglambert can confirm or deny that).

IMO we should sort them in order of our own importance/consideration, but I think the order they're in now is probably good enough to start with?

Reply:
They're not in any particular order